libs/tasks/feature/src/lib/tasks-feature/tasks-feature.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { Task, TasksFacade } from '@todo-workspace/tasks/data-access';
+import { Observable } from 'rxjs';
+import { MatDialog } from '@angular/material/dialog';
+import { TaskFormDialogComponent } from '@todo-workspace/tasks/ui-task-form-dialog';
+
+@Component({
+  selector: 'todo-workspace-tasks-feature',
+  templateUrl: './tasks-feature.component.html',
+  styleUrls: ['./tasks-feature.component.scss'],
```

Comment:
you can sets `ChangeDetection.OnPush` in components for better performance of the application